SIT.Manager/ViewModels/LinuxSettingsPageViewModel.cs

```diff
@@ -0,0 +1,114 @@
+using Avalonia.Platform.Storage;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using SIT.Manager.Interfaces;
+using SIT.Manager.ManagedProcess;
+using SIT.Manager.Models;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SIT.Manager.ViewModels;
+
+public partial class LinuxSettingsPageViewModel : ObservableObject
+{
+    // TODO: Check which services are needed for this ViewModel.
+    private readonly IManagerConfigService _configsService;
+    private readonly IPickerDialogService _pickerDialogService;
+    private readonly IBarNotificationService _barNotificationService;
+    private readonly ILocalizationService _localizationService;
+    
+    [ObservableProperty]
+    private LinuxConfig _config;
+    // DXVK Versions TODO
+    [ObservableProperty] 
+    private List<string> _dxvkVersions;
```

Comment:
`private List<string> _dxvkVersions = []`?

Reply:
I will be adding a way for users to select which version to use:
![image](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/47221652/6abe9fcf-2844-48fe-8e07-2e34bbd1e0d2)
I have not properly implemented this as I think it is less important, but still nice to add in the future